s3pypi/storage.py

```diff
@@ -20,8 +21,17 @@ def __init__(
         s3_endpoint_url: Optional[str] = None,
         s3_put_args: Optional[dict] = None,
         unsafe_s3_website: bool = False,
+        no_sign_request: bool = False,
     ):
-        self.s3 = session.resource("s3", endpoint_url=s3_endpoint_url)
+        _config = None
+        if no_sign_request:
+            _config = Config(signature_version=botocore.session.UNSIGNED)
+
+        self.s3 = session.resource(
+            "s3",
+            endpoint_url=s3_endpoint_url,
+            config=_config,
```

Comment:
This would fit on one line if you removed the trailing comma.

Reply:
of course! Thanks, ill get that one fixed.